pkg/appconsts/global_consts.go

```diff
@@ -67,3 +67,12 @@ func UpgradeHeightDelay() int64 {
 func HashLength() int {
 	return hashLength
 }
+
+// The following consts are not consensus breaking and will be applied straight after this binary is started.
+const (
+	// SdkMsgTransactionCap is the maximum number of SDK messages, aside from PFBs, that a block can contain.
+	SdkMsgTransactionCap = 200
```

Comment:
To make this clearer, can this be just `NonPFBTransactionCap` since PFBs are SDK msgs

Reply:
[5a59b88](https://github.com/celestiaorg/celestia-app/pull/3942/commits/5a59b882b2282460a75d539f3f3675fb8cc597be)